File: src/backend/DevelopersDen/DevelopersDen.Contracts/DBModels/Job/Job.cs
using DevelopersDen.Contracts.DBModels.Recruiter;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DevelopersDen.Contracts.DBModels.Job
{
    public class Job : AuditableEntity
    {
        [Key]
        public Int64 JobId { get; set; }
        [Required]
        public string JobTitle { get; set; }
        [Required]
        public string JobDescription { get; set; }
        public Int32 MinExperience { get; set; }
        [Required]
        public List<string> KeySkills { get; set; }
        [Required]
        public string Location { get; set; }
        [Required]
        public string City { get; set; }
        public DateTime PostedDate { get; set; }
        public Int64 RecruiterAccountId { get; set; }
        public RecruiterAccount RecruiterAccount { get; set; }
    }
}

File: src/backend/DevelopersDen/DevelopersDen.Client/src/app/job-seeker/models/response/job.ts
export interface JobDTO{
    jobs: Job[],
    totalCount: number
}

export interface Job{
    jobId: string,
    jobTitle: string,
    jobDescription: string,
    minExperience: number,
    keySkills: string[],
    location: string,
    city: string,
    jobType: number,
    postedDate: Date,
    recruiter: Recruiter,
    hasApplied: boolean
}

export interface Recruiter{
    companyName: string,
    companyDescription: string,
    websiteUrl: string
}